styles.css

```diff
@@ -0,0 +1,1520 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+@media only screen and (min-width: 767px) {
+  body {
+    margin: 0;
+    padding: 0;
+  }
+
+  html {
+    scroll-behavior: smooth;
+    transition: all 800ms ease;
+  }
+
+  nav {
+    display: flex;
+    width: 100%;
+    background-color: rgb(61, 60, 60);
+    height: 30px;
+    position: fixed;
+    top: 0;
+    z-index: 2;
+  }
+
+  .top-bar {
+    display: flex;
+    align-items: flex-end;
+    justify-content: flex-end;
+    flex-direction: row;
+    top: 0;
+    height: 100%;
+    width: 100vw;
+    z-index: 2;
+    margin-top: 10px;
+    color: rgb(81, 80, 80);
+    font-family: "Lato", sans-serif;
+  }
+
+  .top-bar ul {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    gap: 15px;
+    padding-right: 200px;
+    font-family: 'Lato', sans-serif;
+  }

```

Comment:
As requested by the previous reviewer, please make sure to maintain the navbar links structure in all window sizes 😉. Currently, in small windows, your `nav` links are overlapping and some aren't being displayed. Check the screenshots below:

Your navbar in smaller windows:
![image](https://user-images.githubusercontent.com/42799579/210604126-69550ba5-8aae-4cfc-8ee0-45248824a535.png)

Expected navbar (it should keep its shape):
![image](https://user-images.githubusercontent.com/42799579/210604218-db6a2b2e-fb1b-4ba5-ba89-e9eb2164ac82.png)